### DISCUSSION TITLE
Change objective function

### DISCUSSION THREAD

**Discussion by paocorbin** [opencobra/cobrapy#1135] [Q&A]
Hello, 
We have a model and want to change its objective function, but we can't find a way to make it properly.
The idea is to minimize the difference between measured and predicted growth rate fluxes.
Could you please help us to set this objetive function?
Thank you in advance.

**Comment by cdiener** [comment]
Have a look at how [MOMA is implemented in COBRAPY](https://github.com/opencobra/cobrapy/blob/cc45650ae5acf547a1f419593d727d172cf74502/src/cobra/flux_analysis/moma.py#L52). That is very similar, just that you only minimize the distance for the measured fluxes and you probably want to include a biomass requirement as well.

**Comment by paocorbin** [comment]
Thank you! I will look into it now.
